src/components/Resume/Resume.js

```diff
@@ -0,0 +1,131 @@
+
+                    <p>
+                        mail: mohamad.soufi@outlook.com
+                        <br /> Mobile: +49 157-864-33517 <br /> Adress: Berlin,
+                        Germany
+                        <br /> Social:
+                    </p>
+                    <br />
+
+                    <h2>LANGUAGES</h2>
+
+                    <p>
+                        Native: Arabic
+                        <br /> Fluent: English
+                        <br /> Knowledgeable: German
+                    </p>
+
+                    <br />
+
+                    <h2>STACK</h2>
+
+                    <ul>
+                        <li>React.js, Redux, Vue.js, jQuery, Flexbox, Grid</li>
+                        <li>
+                            Node.js, Express, PostgreSQL, REST, web sockets,{" "}
+                            <br />
+                            Handlebars
+                        </li>
+                        <li>Jest, React Testing Library</li>
+                        <li>Git,GitHub</li>
+                        <li>AWS , Heroku</li>
+                    </ul>
+                </div>
+                <div className="right-side">
+                    <h2>EMPLOYMENT HISTORY</h2>
+                    <br />
+                    <h4>RIDER</h4>
+                    <p>Foodora.Co | 2018 - 2019</p>
+                    <br />
+                    <h2>VOLUNTARY EXPERIENCE</h2>
+                    <h4>DRIVER, WAITER, AND HR</h4>
+                    <p>Soul | Syria | 2009 - 2010</p>
+                    <br />
+                    <h4>FIELD SURVEYOR</h4>
+                    <p>
+                        Humanitarian Relief Foundation | Lebanon | 2012 - 2013
+                    </p>
+                    <br />
+
+                    <h2>EDUCATION HISTORY</h2>
+                    <br />
+                    <h3>
+                        SPICED Academy | Berlin Germany | June - August 2020
+                    </h3>
+                    <ul>
+                        <li>
+                            <a href="/#">Certificate</a>
+                        </li>
+                        <li>
+                            Full stack intensive web development bootcamp using:
+                            <br />- javaScript,React Node,Express ,Postgress and
+                            more .
+                        </li>
+                    </ul>
+                    <br />
+                    <h3>Online courses</h3>
+                    <ul>
+                        <li>
+                            <a href="https://www.udemy.com/course/the-web-developer-bootcamp/">
+                                Web Developer Bootcamp
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.udemy.com/certificate/UC-SOJ0OMQR/">
+                                Certificate
+                            </a>
+                        </li>
```

Comment:
you should either remove this or add it to the li above .. now it looks like a separate course